packages/terra-form-select/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ Unreleased
 ### Changed
 * Updated theme variables
 
+### Added
+* Create dropdown that announces no search results to screen reader
```

Comment:
It looks like the change above has already been released, this entry should be moved into the unreleased section.